### PR TITLE
Updated webscrapper with new fifa_version

### PR DESCRIPTION
**Pull request by kiosia**
Also added lint fixes and generated a new teams json with the updated webscrapper

Related to this issue: https://github.com/alandoni/fifa-champs/issues/142

### CODE REVIEW

File: api/integrations/fifaIndexWebScrapper.js [hunk 1]
```diff
@@ -68,6 +58,10 @@ const TYPE_OF_TEAMS = {
     WOMEN_INTERNATIONAL : 2
 }
 
+/* Since this is a support script,
+we should not be so hard on the no-console rule.*/
+const Console = console;
```
Comment (cristiadu) [inline api/integrations/fifaIndexWebScrapper.js:63]: alright. but why this Console = console ?

File: api/integrations/fifaIndexWebScrapper.js [hunk 2]
```diff
@@ -1,26 +1,16 @@
+#!/usr/bin/env node
```
Comment (cristiadu) [inline api/integrations/fifaIndexWebScrapper.js:1]: why is this here?

File: api/integrations/fifaIndexWebScrapper.js (hunk 1 above)
Comment (cristiadu) [inline api/integrations/fifaIndexWebScrapper.js:63]: alright. but why this Console = console ?

**Review by cristiadu — CHANGES_REQUESTED** [review]
Small comments

File: api/integrations/fifaIndexWebScrapper.js (hunk 2 above)
Comment (kiosia) [reply api/integrations/fifaIndexWebScrapper.js:1]: so we can run this from bash with ./fifaIndexWebScrapper.js

File: api/integrations/fifaIndexWebScrapper.js (hunk 1 above)
Comment (kiosia) [reply api/integrations/fifaIndexWebScrapper.js:63]: This way, lint stops poiting the console use as an error. This is the recommended way when you are using console to output info for the user, which is the case

Comment (kiosia) [reply api/integrations/fifaIndexWebScrapper.js:63]: YOU ALREADY ASKED THIS BRO

Comment (cristiadu) [reply api/integrations/fifaIndexWebScrapper.js:63]: My internet yesterday was terrible :/
